src/gsy_e/models/strategy/forward/load.py

```diff
@@ -33,17 +44,31 @@ def state(self) -> "LoadState":
     def event_activate(self, **kwargs):
         self._energy_params.event_activate_energy(self.area)
 
+    def remove_order(self, market: "ForwardMarketBase", market_slot: DateTime, order_uuid: str):
+        bids = [bid
+                for bid in market.slot_bid_mapping[market_slot]
+                if bid.buyer == self.owner.name and bid.id == order_uuid]
+        if not bids:
+            self.log.error("Bid with id %s does not exist on the market %s %s.",
+                           order_uuid, market.market_type, market_slot)
+        for bid in bids:
+            market.delete_bid(bid)
+
     def remove_open_orders(self, market: "ForwardMarketBase", market_slot: DateTime):
         bids = [bid
                 for bid in market.slot_bid_mapping[market_slot]
                 if bid.buyer == self.owner.name]
         for bid in bids:
             market.delete_bid(bid)
 
```

Comment:
Maybe you could add an optional parameter `oder_uuid` here and re-use this method in `remove_order`

Reply:
This was a mistake on my part, since it is not that reusable at the end. The PV needs to check the offer.seller and the Load the bid.buyer, and use the market.slot_offer_mapping and market.slot_bid_mapping respectively. I have adapted the tests already in one of the previous comments in order to catch this error. Thanks!